remo/base/views.py

```diff
@@ -42,8 +43,16 @@ def login_failure(self, msg=''):
 def main(request):
     """Main page of the website."""
     featured_rep = utils.latest_object_or_none(FeaturedRep)
-    feed = feedparser.parse(settings.PLANET_URL)
-    planet_entries = feed.entries[:3]
+
+    res = False
+    try:
```

Comment:
No need to ``GET`` two times the same thing (keep in mind it is the homepage).
We can always wrap ``feedparser.parse`` with a timeout check instead.